v1/openapi/schemas/request.yaml

```diff
@@ -0,0 +1,74 @@
+x-component: "schemas/request"
+description: Request representation.
+type: object
+properties:
```

Comment:
At documentation we also have `is_original_request` field, did we forgot to add it here or delete it in docs?

Reply:
Add